lunamark/writer/generic.lua

```diff
@@ -351,6 +351,17 @@ function M.new(options)
     return ""
   end
 
+  --- A line block.
+  -- The indentation spaces are changed into nbsp (00A0) as with Pandoc.
+  function W.lineblocks(lines)
```

Comment:
An after though, wondering if this ought to be singular here. It's a (single) block of (several) lines.

Reply:
Done